package.json

```diff
@@ -30,34 +30,38 @@
     "@babel/core": "^7.1.6",
     "@babel/preset-env": "^7.1.6",
     "@babel/preset-flow": "^7.0.0",
-    "babel-eslint": "^10.0.2",
     "@babel/preset-react": "^7.0.0",
-    "babel-loader": "^8.0.4",
-    "@bigtest/interactor": "^0.7.0",
-    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/interactor": "^0.9.3",
     "@bigtest/mocha": "^0.5.1",
     "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^3.2.1",
     "@folio/stripes": "^4.0.0",
     "@folio/stripes-cli": "^1.13.0",
     "@folio/stripes-core": "^5.0.1",
     "@folio/stripes-react-hotkeys": "^3.0.6",
+    "babel-eslint": "^10.0.2",
+    "babel-loader": "^8.0.4",
+    "babel-polyfill": "^6.26.0",
     "chai": "^4.2.0",
     "eslint": "^6.2.1",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
     "eslint-loader": "^2.1.2",
     "eslint-plugin-flowtype": "^3.4.2",
     "eslint-plugin-prettier": "^3.0.1",
     "flow-bin": "^0.98.0",
     "flow-typed": "^2.5.1",
     "husky": "^1.3.1",
     "immutable": "^3.7.5",
+    "inflected": "^2.0.4",
     "lodash": "^4.17.4",
+    "miragejs": "^0.1.40",
     "mocha": "^5.2.0",
     "moment": "^2.22.2",
     "qs": "^6.5.1",
+    "react": "^16.5.0",
+    "react-dom": "^16.5.0",
+    "react-intl": "^5.3.2",
```

Comment:
We're only on v4.5 at present. Also, they only published `v4.0.0` less than six months ago, and `v5.0.0` less than a month ago and they're already on v5.4!!! So much churn; ugh.

Reply:
(Sorry; just noticed this now; I know you've been through many versions of this PR. I'm not trying to kill you with 1000 small cuts, promise.)

Reply:
> Sorry; just noticed this now; I know you've been through many versions of this PR. I'm not trying to kill you with 1000 small cuts, promise.

No worries 👌 

I believe i added those dependencies to remove some of the errors from `yarn install` but I went ahead and downgraded them to the same version as what's listed in peerDependency.